tests/framework/installer/ceph_helm_installer.go

```diff
@@ -117,8 +117,9 @@ func (h *CephInstaller) configureRookCephClusterViaHelm(upgrade bool) error {
 	values["configOverride"] = clusterCustomSettings
 	values["toolbox"] = map[string]interface{}{
 		"enabled":   true,
-		"image":     "rook/ceph:" + h.settings.RookVersion,
 		"resources": nil,
+		// Use default image
+		"image":     "",
```

Comment:
I wonder if we need to remove this property completely. It must be picking up this empty string instead of allowing the default value to be set.

Reply:
```suggestion
```